newspaper/article.py

```diff
@@ -10,6 +10,7 @@
 import glob
 
 import requests
+from langdetect import detect
```

Comment:
`detect` is easy to overload, can we `import langdetect` and use `langdetect.detect(..)`?